setup.cfg

```diff
@@ -27,6 +27,7 @@ package_dir =
     = src
 packages = find_namespace:
 install_requires =
+    pytz ~= 2021.1
```

Comment:
Do we need to add this dependency? Can we create a UTC time just using `datetime`?

Reply:
I was wondering this myself. I read that pytz is required to get full timezone coverage, but `datetime.now(tz=timezone.utc)` seems to work for me. Given this is a template library, I agree that we can remove this dependency and have others add it if they encounter a datetime edge case.